ui/pages/settings/networks-tab/index.scss

```diff
@@ -254,6 +254,7 @@
     text-overflow: ellipsis;
     white-space: nowrap;
     overflow: hidden;
+    flex-grow: 1;
 
     &:hover {
       cursor: pointer;
```

Comment:
Giving the network page a bit of a facelift

### Before

<img width="940" alt="Screen Shot 2022-05-19 at 3 55 03 PM" src="https://user-images.githubusercontent.com/8112138/169418817-68ba3a4b-1de0-4e1c-a4d9-106bf1d3291b.png">

### After

<img width="994" alt="Screen Shot 2022-05-19 at 3 52 25 PM" src="https://user-images.githubusercontent.com/8112138/169418839-558199d2-1bc9-48a1-812c-bd87a04b085e.png">